Rakefile

```diff
@@ -89,6 +89,37 @@ task :test do
   sh 'cargo test --workspace'
 end
 
+namespace :unicode do
+  ucd_dir = 'generated/ucd'
+
+  desc 'Rebuild Rust generated Rust sources from Unicode data'
+  task :build do
+    unless system('ucd-generate --version')
+      puts 'Please install ucd-generate > 0.3.0 first'
+      exit(1)
+    end
```

Comment:
An older version of `ucd-generate` won't be caught by this. Let me actually check the installed version too.

Reply:
Fixed in 2bbedfb.

Error for `ucd-generate` not installed:
> <img width="770" alt="error ucd-generate not installed" src="https://github.com/artichoke/roe/assets/12410942/109cf1d8-102e-46b3-95e8-c6c9dc072b86">

Error for older version of `ucd-generate`:
> <img alt="error please upgrade ucd-generate" src="https://github.com/artichoke/roe/assets/12410942/81ccc30a-4794-4393-a753-cb99ff8a7e64">